### PR TITLE
Issue #11719: Activate all group for pitest-regexp

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
#11719
Activating `ALL` group for `pitest-regexp`

Every mutator has been activated except `INLINE_CONSTS`

- Report with `ALL` group except `INLINE_CONSTS`: https://vyom-yadav.github.io/pitest-all-latest/pitest-regexp/index.html

### CODE REVIEW

File: .ci/pitest-suppressions/pitest-regexp-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>RegexpMultilineCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.regexp.RegexpMultilineCheck</mutatedClass>
+    <mutatedMethod>getRegexCompileFlags</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.ReturnValuesMutator</mutator>
+    <description>replaced return of primitive boolean/byte/short/integer value with (x == 1) ? 0 : x + 1</description>
+    <lineContent>return result;</lineContent>
+  </mutation>
```
Comment (nick-mancuso) [inline .ci/pitest-suppressions/pitest-regexp-suppressions.xml:64]: @Vyom-Yadav please open PR with hardcoded mutation for this one.

**Review by nick-mancuso — CHANGES_REQUESTED** [review]
Item:

File: .ci/pitest-suppressions/pitest-regexp-suppressions.xml (hunk 1 above)
Comment (Vyom-Yadav) [reply .ci/pitest-suppressions/pitest-regexp-suppressions.xml:64]: #11999, I won't be adding comment explanation comment at https://github.com/checkstyle/checkstyle/issues/11719, this particular mutant is very well documented at https://pitest.org/quickstart/mutators/#RETURN_VALS

**Review by nick-mancuso — APPROVED** [review]

**Comment by nick-mancuso** [comment]
Failure in CI at https://checkstyle.semaphoreci.com/workflows/53a1078a-8744-49f2-b3dc-44dc330aee24?pipeline_id=2ba79a2c-a822-49b2-b733-c6e1b9d735bf is not related to changes here.

**Pull request merged**